test/test_url_validity.py

```diff
@@ -1,15 +1,19 @@
 #!/usr/bin/env python
 
+
+
 from __future__ import print_function
 
 
-from io import StringIO
+try:
+    from cStringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
Since the script is currently incompatible with Python 3 anyway the exception handling is not useful.